samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -1253,7 +1253,10 @@
       "Definition": {
         "Statement": [
           {
-            "Action": "kms:Encrypt",
+            "Action": [
```

Comment:
Can we add some unit tests for this? You can follow this guide: https://github.com/aws/serverless-application-model/blob/develop/DEVELOPMENT_GUIDE.md#transform-tests